fifo-sns-sqs-lambda-firehose-s3/modules/terraform-aws-firehose/main.tf

```diff
@@ -18,10 +18,10 @@ resource "aws_kinesis_firehose_delivery_stream" "stream" {
 
 
   extended_s3_configuration {
-    role_arn        = aws_iam_role.firehose_role.arn
-    bucket_arn      = var.bucket_arn
-    buffer_size     = var.buffer_size != null ? var.buffer_size : 1
-    buffer_interval = var.buffer_interval != null ? var.buffer_interval : 60
+    role_arn           = aws_iam_role.firehose_role.arn
+    bucket_arn         = var.bucket_arn
+    buffering_size     = var.buffer_size != null ? var.buffer_size : 1
+    buffering_interval = var.buffer_interval != null ? var.buffer_interval : 60
```

Comment:
note: Update `s3_configuration` args for AWS Provider v5.

- v5
    - https://registry.terraform.io/providers/hashicorp/aws/5.100.0/docs/resources/kinesis_firehose_delivery_stream
- v4
    - https://registry.terraform.io/providers/hashicorp/aws/4.67.0/docs/resources/kinesis_firehose_delivery_stream